tests/track_blockmatching_test.py

```diff
@@ -0,0 +1,21 @@
+import cv2
+import numpy as np
+import sys
+from os.path import dirname, abspath
+
+sys.path.append(dirname(dirname(abspath(__file__))))
+
+from track_blockmatching import track_frame
+
+def test_track_frame():
+    cap = cv2.VideoCapture('tests/data/2019-07-12 circumnutation.mp4')
+    ret, photo0 = cap.read()
+    ret, photo1 = cap.read()
```

Comment:
`photo0` should be read from `tests/data/frame_0001.jpg`
`photo1` should be read from `tests/data/frame_0002.jpg`

Reply:
This code is duplicated from `test_api_track_frame`; We should probably have a function that returns `photo0,photo1` and is called by both.